ui/src/app/components/add-new-column/add-new-column.component.ts

```diff
@@ -19,9 +21,17 @@ export class AddNewColumnComponent implements OnInit {
   tableId: string = ""
   selectedNull: boolean = true
   dataTypesWithColLen: string[] = ColLength.DataTypes
+  autoGenMap : any = {}
+  selectedAutoGen: AutoGen = {
+    Name: '',
+    GenerationType: ''
+  }
+  processedAutoGenMap: GroupedAutoGens = {};
+  srcDbName: String = localStorage.getItem(StorageKeys.SourceDbName) as string
```

Comment:
Is there another reference where we have called `localStorage` here? I have usually seen these values being set and unset in `ngOnInit()`. Please check other references in the code once.

Reply:
In workspace we fetch this value in this way already. Thats the code I referenced